backend/src/test/java/site/coduo/acceptance/PairRoomHistoryAcceptanceTest.java

```diff
@@ -44,7 +45,8 @@ static void savePairRoomHistory(final String accessCode, final PairRoomHistoryCr
     @DisplayName("페어룸 히스토리를 저장한다.")
     void create_pair_room_history() {
         // given
-        final PairRoomCreateResponse pairRoomCreateResponse = createPairRoom(new PairRoomCreateRequest("해시", "파란"));
+        final PairRoomCreateResponse pairRoomCreateResponse =
+                createPairRoom(new PairRoomCreateRequest("해시", "파란", PairRoomStatus.ONBOARDING.name()));
```

Comment:
[질문] 
사소할  수 있지만, 이유가 있는지 알고 싶어 질문합니다.  CategoryAcceptanceTest에서는 아래와 같은 스타일로 하셨는데,
여기선 스타일을 변경하신 이유가 무엇인가요?

```suggestion
                createPairRoom(new PairRoomCreateRequest("해시", "파란", "ONBOARDING"));
```

Reply:
처음에는 문자열을 넣어줬다가, 나중에 변수를 넣는 방식으로 바꿨는데 이 부분은 놓쳤네요!
꼼꼼한 피드백 감사드립니다!